src/models/reader.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (connection, DataTypes) => {
+  const schema = {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: { args: true, msg: "Please enter a name" },
+        notNull: { args: true, msg: "Please enter a name" },
+      },
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        isEmail: { args: true, msg: "use a valid email address" },
+        notEmpty: { args: true, msg: "Please enter an email" },
+        notNull: { args: true, msg: "Please enter an email" },
+      },
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: { args: [8, 16], msg: "password too short or long" },
+        notEmpty: { args: true, msg: "Please enter a password" },
+        notNull: { args: true, msg: "Please enter a password" },
+      },
+    },
+    books: DataTypes.STRING
```

Comment:
I don't think this field is doing anything as data would have to be manually input into it. The books will be associated with a reader when the ReaderId is updated on a particular book, then a list of books will be shown if a get request is sent to readers. This is because you have used "eager loading" by adding the sequelize association hasMany in the index.js